components/schemes/Typography/src/MDCTypographyScheme.m

```diff
@@ -63,6 +65,14 @@ - (instancetype)initWithDefaults:(MDCTypographySchemeDefaults)defaults {
 #endif
         break;
     }
+
+    if (defaults == MDCTypographySchemeDefaultsMaterial201902) {
```

Comment:
Please move this into the `switch` statement above.  It's less likely to cause problems if we don't spread the logic out into different sections.